pyproject.toml

```diff
@@ -1,8 +1,23 @@
 [tool.poetry]
-name = "usn"
-version = "0.1.0"
+name = "ubuntu-server-netboot"
+packages = [
+    {include = "usn"}
+]
+version = "0.1.1rc3"
 description = "This utility generates a netboot directory tree from an Ubuntu Server Live ISO image, an image based on the subiquity installer."
 authors = ["dann frazier <dannf@ubuntu.com>", "Taihsiang Ho (tai271828) <taihsiangho@ubuntu.com>"]
+license = "GPL-2.0-only"
+readme = "README.md"
+classifiers = [
+    "Topic :: Internet :: File Transfer Protocol (FTP)",
```

Comment:
I'm not sure FTP is really a good fit - what about "Topic :: System :: Installation/Setup" maybe?

Reply:
`Installation/Setup` is way much better. I did not notice this option. Thanks for heading up! The pull request is updated and published as 0.1.1rc4. See this page to check what it looks like https://test.pypi.org/project/ubuntu-server-netboot/0.1.1rc4/